rootfs/rootfs/etc/rc.d/install_ca_certs

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+BOOT2DOCKER_CERTS_DIR=/var/lib/boot2docker/certs
+CERTS_DIR=/etc/ssl/certs
+CAFILE=${CERTS_DIR}/ca-certificates.crt
+ 
+for cert in $(/bin/ls -1 ${BOOT2DOCKER_CERTS_DIR}); do
```

Comment:
Scripting the output of `ls` is generally considered dangerous, so I'd prefer to see this use something like:
```bash
for cert in "$BOOT2DOCKER_CERTS_DIR"/*; do
```